x-pack/legacy/plugins/siem/public/pages/hosts/index.tsx

```diff
@@ -16,8 +18,28 @@ import { Hosts } from './hosts';
 export const HostsContainer = pure<HostComponentProps>(({ match }) => (
   <>
     <Switch>
-      <Route strict exact path={match.url} component={Hosts} />
-      <Route path={`${match.url}/:hostName`} component={HostDetails} />
+      <Route
+        strict
+        exact
+        path={match.url}
+        render={props => (
+          <PageRoute
+            {...props}
+            component={Hosts}
+            title={i18n.translate('xpack.siem.pages.Hosts.HostsTitle', {
```

Comment:
For the most part our translation keys have been lower case separated by dots:

Examples:
```ts
xpack.siem.hosts.kqlPlaceholder
xpack.siem.navigation.overview
```

Should these just be `xpack.siem.pages.hosts.hostsTitle` with camel case?

Reply:
Yes, thanks for pointing this out.